data_validation/schema_validation.py

```diff
@@ -172,3 +200,96 @@ def schema_validation_matching(source_fields, target_fields, exclusion_fields):
                 ]
             )
     return results
+
+def is_number(val):
```

Comment:
Python has a _string_.isdigit() method that also returns a boolean.

Reply:
done